internal/pkg/cache/cache.go

```diff
@@ -43,7 +43,7 @@ func PutObject(identifier string, data []byte) error {
 		return ErrorInvalidCacheIdentifier
 	}
 
-	err := os.MkdirAll(cacheFolderPath, os.ModePerm)
+	err := os.MkdirAll(cacheFolderPath, 0o750)
```

Comment:
What is the linting error here? Merely curiosity

Reply:
internal/pkg/cache/cache.go:46:9: G301: Expect directory permissions to be 0750 or less (gosec)
	err := os.MkdirAll(cacheFolderPath, os.ModePerm)
	
ModePerm seems to set it to 777